src/Microsoft.Restier.Security/ApiPolicyActivator.cs

```diff
@@ -50,10 +50,10 @@ public Expression Expand(QueryExpressionContext context)
                 return CallInner(context);
             }
 
-            var target = context.QueryContext.GetApiService<ApiBase>();
-            var entitySetProperty = target.GetType().GetProperties(
+            var apiTypes = context.QueryContext.GetApiService<ApiTypeInfo>().ApiTypes;
```

Comment:
This is only to show the usage of ApiTypeInfo, each instance of ApiPolicyActivator should still be applied to a specific Api type.